app/views/activities/_activity.html.erb

```diff
@@ -1,7 +1,15 @@
-<a class="list-group-item t-activity" href="<%= edit_appointment_path(activity.appointment) %>">
-  <span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>
-  <b><%= activity.user_name %></b>
-  <%= content_for(:activity_message) %>
-  <%= render('activities/details', appointment: activity.appointment) if details %>
-  <em class="badge"><%= activity.created_at.to_s(:govuk_date_short) %> (<%= time_ago_in_words(activity.created_at)%> ago)</em>
-</a>
+<% if link %>
+  <a class="list-group-item t-activity" href="<%= edit_appointment_path(activity.appointment) %>">
+<% else %>
+  <li class="list-group-item t-activity">
+<% end %>
+<span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>
```

Comment:
Can we indent these lines as if they're in the tag? Might make it a bit more readable.